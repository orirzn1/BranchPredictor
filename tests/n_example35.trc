4 5 9 2 global_history local_tables using_share_mid
0xd808d0 T 0x10dc
0x828076e0 N 0x3cdfeb28
0x828076e0 N 0x3cdfeb28
0xc87faad4 N 0x17c149c
0x9dc N 0xec27674
0xc87faad4 N 0x17c149c
0xc87faad4 T 0x17c149c
0xc87faad4 N 0x17c149c
0x9dc T 0xec27674
0xd808d0 T 0x10dc
0x828076e0 T 0x3cdfeb28
0xc87faad4 N 0x17c149c
0xc87faad4 T 0x17c149c
0xc87faad4 T 0x17c149c
0xd808d0 T 0x10dc
0xd808d0 N 0x10dc
0x9dc N 0xec27674
0xd808d0 N 0x10dc
0xc87faad4 T 0x17c149c
0x828076e0 T 0x3cdfeb28
0x9dc T 0xec27674
0xd808d0 N 0x10dc
0x9dc N 0xec27674
0xd808d0 T 0x10dc
0x828076e0 N 0x3cdfeb28
0xd808d0 T 0x10dc
0x828076e0 N 0x3cdfeb28
0x9dc N 0xec27674