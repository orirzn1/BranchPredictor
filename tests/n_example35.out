0xd808d0 N 0xd808d4
0x828076e0 N 0x828076e4
0x828076e0 T 0x3cdfeb28
0xc87faad4 N 0xc87faad8
0x9dc N 0x9e0
0xc87faad4 T 0x17c149c
0xc87faad4 T 0x17c149c
0xc87faad4 T 0x17c149c
0x9dc T 0xec27674
0xd808d0 N 0xd808d4
0x828076e0 N 0x828076e4
0xc87faad4 T 0x17c149c
0xc87faad4 T 0x17c149c
0xc87faad4 T 0x17c149c
0xd808d0 N 0xd808d4
0xd808d0 T 0x10dc
0x9dc T 0xec27674
0xd808d0 T 0x10dc
0xc87faad4 T 0x17c149c
0x828076e0 N 0x828076e4
0x9dc T 0xec27674
0xd808d0 N 0xd808d4
0x9dc N 0x9e0
0xd808d0 T 0x10dc
0x828076e0 N 0x828076e4
0xd808d0 N 0xd808d4
0x828076e0 N 0x828076e4
0x9dc T 0xec27674
flush_num: 14, br_num: 28, size: 421b
